### PR TITLE
Make scripts Linux compliant

### PR DESCRIPTION
**Pull request by bemug**

### CODE REVIEW

File: deploy_webserver.cmd [hunk 1]
```diff
@@ -1,2 +0,0 @@
-cd web
-npm install .
```
Comment (Telsanyr) [inline deploy_webserver.cmd:2]: Why did you delete this, my friend ? It should have been renamed like you did with the other script files.

**Review by Telsanyr — CHANGES_REQUESTED** [review]
Not Today

File: deploy_webserver.cmd (hunk 1 above)
Comment (bemug) [reply deploy_webserver.cmd:2]: This is just git's behavior. The squashed commit thinks it's another files, but in commit history it's not.

Comment (Telsanyr) [reply deploy_webserver.cmd:2]: Ok

**Review by Telsanyr — APPROVED** [review]
